src/components/LoginForm/styles.js

```diff
@@ -0,0 +1,83 @@
+import { StyleSheet } from 'react-native';
+
+export default StyleSheet.create({
+
+    ViewStyle: {
+        backgroundColor: "#dbdbdb",
+        width: 300,
```

Comment:
OK, now here is the thing. If your are defining kind of device related styles such as screen width, height, better to do that in % way. Otherwise we will see different different views in different different devices which is not good.

Reply:
I have updated the values of the width to percentage, but height cannot be changed to percentage because this makes text input very small and unreadable and this displaces the Login/Signup button as well! So I had to set the height value as it is now. I have tested this in different emulators too, ranging from 4" to 6.2" and this adjusts well. I hope it is fine!